src/sentry/static/sentry/app/components/group/inboxBadges/inboxReason.tsx

```diff
@@ -30,6 +30,42 @@ function InboxReason({inbox, fontSize = 'sm'}: Props) {
       ? `More than ${Math.round(count / EVENT_ROUND_LIMIT)}k`
       : `${count}`;
 
+  const getToolTipDescription = ({
+    until,
+    count,
+    window,
+    user_count,
+    user_window,
+  }: InboxReasonDetails) => {
+    if (until) {
+      // Was ignored until `until` has passed.
+      //`until` format "2021-01-20T03:59:03+00:00"
+      return t('Was ignored until %(window)s', {
+        until,
+      });
+    } else if (count) {
+      // Was ignored until `count` events occured
+      // If `window` is defined, than `count` events occurred in `window` minutes.
+      // else `count` events occured since it was ignored.
+      return t('Was ignored until it occured %(count)s time(s)%(window)s', {
+        count: getCountText(count || 0),
+        window: window ? ' in ' + getDuration((window || 0) * 60, 0, true) : '',
```

Comment:
should we translate 'in' too?

Reply:
I'm not sure how the translator works. I was hoping `in` _would_ get translated like this. Otherwise it gets a little more cumbersome because `in` is only present in the string if window exists.

Reply:
we could write as below:
```

   if (count) {
      // Was ignored until `count` events occured
      // If `window` is defined, than `count` events occurred in `window` minutes.
      // else `count` events occurred since it was ignored.
      if (window) {
        return tct('Was ignored until it occured ([count]) time(s) in [duration]', {
          count: getCountText(count || 0),
          duration: getDuration((window || 0) * 60, 0, true),
        });
      }

      return tct('Was ignored until it occured ([count]) time(s)', {
        count: getCountText(count || 0),
      });
    }
```

Reply:
Alright, wrote it like this so that translations work better :)